python/OpenMCInput.py

```diff
@@ -9,6 +9,8 @@
 from OpenMCSurface import SurfaceCard,surface_from_attribute, write_openmc_surface
 from OpenMCCell import cell_from_attribute, write_openmc_cell
 from OpenMCMaterial import material_from_attribute, write_openmc_material
+from OpenMCLattice import lattice_from_attribute, write_openmc_lattice
```

Comment:
@makeclean Andrew, I'm interested in helping with building out the functionality in this branch. I have an interest in supporting translation back and forth between MCNP and  OpenMC. I am curious though, if this addition is just a placeholder, or if you happen to have an uncommitted OpenMCLattice.py file (there doesn't seem to be one in this branch). Thanks.

Reply:
Hi @rtswanson I think I have a partial file somewhere, I'll push it and then feel free to work from that. I basically hit a wall that I didn't have the bandwidth for, at my inspection from the MCNP manual and the https://openmc.readthedocs.io/en/stable/io_formats/geometry.html#lattice-element OpenMC - I couldnt see an easy 1-1 mapping. But I would be thrilled if you want to take this on :)